addons/web/static/src/views/sample_server.js

```diff
@@ -467,7 +467,11 @@ export class SampleServer {
         const measures = [];
```

Comment:
commit message seems to be outdated and not correspond to what is done in commit

Reply:
Another fix has been done, so indeed the commit is a bit outdated. 😕